app/assets/javascripts/discourse/app/components/composer-editor.js

```diff
@@ -305,6 +305,14 @@ export default Component.extend(ComposerUploadUppy, {
     }
   },
 
+  _newTopicState() {
```

Comment:
this should be a getter instead. and perhaps named `_isNewTopic` ("state" is ambiguous)

Reply:
Ah yes good point, just pushed a change, thanks!